core-bundle/src/Resources/contao/dca/tl_page.php

```diff
@@ -24,7 +24,7 @@
 			array('tl_page', 'setRootType'),
 			array('tl_page', 'showFallbackWarning'),
 			array('tl_page', 'makeRedirectPageMandatory'),
-			array('tl_page', 'generateSitemap')
+			array('tl_page', 'generateSitemap'),
```

Comment:
Why this?

Reply:
That's just a leftover, since I had an additional callback there before. Interestingly this is not catched by code style fixer?

Reply:
That is because there is no strict rule not to have a trailing comma in a multi-line array.